runtime/dart_isolate.cc

```diff
@@ -1117,14 +1276,25 @@ void DartIsolate::OnShutdownCallback() {
   }
 }
 
+void DartIsolate::OnMessageEpilogue(Dart_Handle result) {
+  if (is_platform_isolate_ && Dart_CurrentIsolate() != nullptr) {
```

Comment:
If you get a `Dart_Handle` as parameter here the current thread should be entered in an isolate and therefore imply `FML_CHECK(Dart_CurrentIsolate() != nullptr)`, no?

Reply:
Ah, I remember why I added this now. This case sometimes happens in one of the unit tests: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8755438309850567809/+/u/test:_test:_Host_Tests_for_host_debug_unopt/stdout